reporter-web-app/src/reducers/selectors.js

```diff
@@ -73,8 +73,8 @@ const getTotalNrLicenses = (projectsLicenses) => {
         return accumulator;
     }, licensesSet).size;
 };
-const hasReportDataChanged = (newArg, lastArg) => newArg.data.reportLastUpdate === lastArg.data.reportLastUpdate;
-
+const hasReportDataChanged = (newArgs, oldArgs) => newArgs.length === oldArgs.length
+    && newArgs[0].data.reportLastUpdate === oldArgs[0].data.reportLastUpdate;
```

Comment:
Isn't the logic inverted here? The variable is called "has changed", but you check for equality here.

Reply:
Let me investigate

Reply:
Fixed it in https://github.com/heremaps/oss-review-toolkit/pull/1284/commits/73940e542fe2d9f79431ecf63cadfa47c98f8495